### PR TITLE
Package srt.0.3.0

### PR DESCRIPTION
**Pull request by toots**
### `srt.0.3.0`
Binding for the Secure, Reliable, Transport protocol library
Secure Reliable Transport (SRT) is an open source transport technology
that optimizes streaming performance across unpredictable networks, such
as the Internet.
This package provides OCaml bindings to the C implementation library.

---
* Homepage: https://github.com/savonet/ocaml-srt
* Source repo: git+https://github.com/savonet/ocaml-srt.git
* Bug tracker: https://github.com/savonet/ocaml-srt/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

**Comment by mseri** [comment]
The failure to build `conf-srt-openssl` in debian (and thus this package) is expected?

File: packages/srt/srt.0.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+synopsis: "Binding for the Secure, Reliable, Transport protocol library"
+description: """
+Secure Reliable Transport (SRT) is an open source transport technology
+that optimizes streaming performance across unpredictable networks, such
+as the Internet.
+This package provides OCaml bindings to the C implementation library.
+"""
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+license: "GPL-2.0"
```
Comment (mseri) [inline packages/srt/srt.0.3.0/opam:12]:
```suggestion
license: "GPL-2.0-only"
```
or
```suggestion
license: "GPL-2.0-or-later"
```
?

**Comment by toots** [comment]
> The failure to build `conf-srt-openssl` in debian (and thus this package) is expected?

Not it's not but it's also most likely a ubuntu bug. I suspect `libsrt-dev` does not install the corresponding SSL dev package. This package could be from gnutls or openssl so it should really be `libsrt-dev`'s responsibility to install the one it's been compiled with.

**Comment by toots** [comment]
License clarification backported upstream thanks!

File: packages/srt/srt.0.3.0/opam [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+synopsis: "Binding for the Secure, Reliable, Transport protocol library"
+description: """
+Secure Reliable Transport (SRT) is an open source transport technology
+that optimizes streaming performance across unpredictable networks, such
+as the Internet.
+This package provides OCaml bindings to the C implementation library.
+"""
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+license: "GPL-2.0-only"
```
Comment (kit-ty-kate) [inline packages/srt/srt.0.3.0/opam:12]: Note that according to your files (e.g. https://github.com/savonet/ocaml-srt/blob/a10d0785c188236c3c11e70d53f4ce9a021f09e1/src/srt.mli#L9) this should be `GPL-2.0-or-later` instead

**Comment by kit-ty-kate** [comment]
Thanks

**Comment by toots** [comment]
License fixed again. Thanks for the attention to details y'all!

**Pull request merged**